build.sbt

```diff
@@ -63,21 +63,32 @@ lazy val `scala-native-jdbc-sqlite` = project
   .enablePlugins(ScalaNativePlugin)
   .dependsOn(`scala-native-jdbc`)
 
-lazy val `scala-native-jdbc-tests` = crossProject(JVMPlatform, NativePlatform)
+
+lazy val `scala-native-jdbc-duckdb` = project
+  .in(file("scala-native-jdbc-duckdb"))
+  .settings(
+    name := "scala-native-jdbc-duckdb"
+  )
+  .enablePlugins(ScalaNativePlugin)
+  .dependsOn(`scala-native-jdbc`)
+
+lazy val `scala-native-jdbc-sqlite-tests` = crossProject(JVMPlatform, NativePlatform)
   .crossType(CrossType.Pure)
-  .in(file("scala-native-jdbc-tests"))
+  .in(file("scala-native-jdbc-sqlite-tests"))
   .settings(
-    name := "scala-native-jdbc-tests",
+    name := "scala-native-jdbc-sqlite-tests",
     noPublishSettings,
     libraryDependencies ++= Seq(
       "org.scalameta" %%% "munit" % "1.0.0" % Test
     )
   )
   .nativeSettings(
-    nativeConfig ~= {
-      _.withServiceProviders(
+    nativeConfig ~= {c =>
+      c.withServiceProviders(
         Map(
-          "java.sql.Driver" -> Seq("com.github.lolgab.jdbc.sqlite.SQLiteDriver")
+          "java.sql.Driver" -> Seq(
+            "com.github.lolgab.jdbc.sqlite.SQLiteDriver",
+          )
```

Comment:
I'm starting to think we should add the `@EnableReflectiveInstantiation` to the Driver classes so users don't need to add them to reflection... After all if you add a certain library it means you want to use it, and the library is already modularized, so you don't need to add sqlite if you use duckdb, and viceversa.
But it's a change that can be done in a later PR.

Reply:
> But it's a change that can be done in a later PR.

I'll leave this to you, if you don't mind, because I'm really unfamiliar with `@EnableReflectiveInstantiation` (and reflection in general) 😅